src/main/java/com/piekoszek/nowaksharedrent/invite/Invitation.java

```diff
@@ -0,0 +1,15 @@
+package com.piekoszek.nowaksharedrent.invite;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@Getter
+public class Invitation {
```

Comment:
has to be public?

Reply:
I have some issues with dtos. Sometimes I'm getting priviligaes exception even though dto is used inside on package. I will try to change it. Maybe this time it will be okay.

Reply:
As discussed on discord. I am keeping it public